From e5e168ac1e9ed9ae5653ce95e8147ecd60a42c9f Mon Sep 17 00:00:00 2001
From: Akshay Jaggi <akshay1994.leo@gmail.com>
Date: Mon, 28 Sep 2015 19:06:36 +0530
Subject: [PATCH] Patch

---
 Makefile.am | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index fcf6805..8c082a2 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -49,18 +49,3 @@ endif
 if USE_DNS_SD
 shairport_sync_SOURCES += mdns_dns_sd.c
 endif
-
-install-exec-hook:
-if INSTALL_CONFIG_FILES
-	[ -e $(DESTDIR)/etc ] || mkdir $(DESTDIR)/etc
-	cp scripts/shairport-sync.conf $(DESTDIR)/etc/shairport-sync.conf.sample
-	[ -f $(DESTDIR)/etc/shairport-sync.conf ] || cp scripts/shairport-sync.conf $(DESTDIR)/etc/shairport-sync.conf
-endif
-if INSTALL_SYSTEMV
-	[ -e $(DESTDIR)/etc/init.d ] || mkdir -p $(DESTDIR)/etc/init.d
-	[ -f $(DESTDIR)/etc/init.d/shairport-sync ] || cp scripts/shairport-sync $(DESTDIR)/etc/init.d/
-endif
-if INSTALL_SYSTEMD
-	[ -e $(DESTDIR)/usr/lib/systemd/system ] || mkdir -p $(DESTDIR)/usr/lib/systemd/system
-	cp scripts/shairport-sync.service $(DESTDIR)/usr/lib/systemd/system
-endif
-- 
2.5.0

