diff -aur shairport-sync/Makefile.am shairport-sync.patched/Makefile.am
--- shairport-sync/Makefile.am	2014-11-16 23:06:34.129763472 +0100
+++ shairport-sync.patched/Makefile.am	2014-11-16 23:07:53.849732815 +0100
@@ -35,7 +35,3 @@
 if USE_DNS_SD
 shairport_sync_SOURCES += mdns_dns_sd.c
 endif
-
-install-exec-hook:
-	[ -f /etc/init.d/shairport-sync ] || cp scripts/shairport-sync /etc/init.d/
-	update-rc.d shairport-sync defaults 90 10
